discovery/discovery.go

```diff
@@ -5,6 +5,7 @@ package discovery
 import (
 	"bytes"
 	"fmt"
+	"github.com/dtn7/dtn7-go/core"
```

Comment:
This may be nitpicking, but do you like to move this include to the lower ones? In other words, that the packages of the standard library and the external ones are grouped.